apps/client/app/goods-receipts/[id]/page.tsx

```diff
@@ -0,0 +1,29 @@
+// app/goods-receipts/[id]/page.tsx
+import GoodsReceiptForm from '../../components/GoodsReceiptForm';
+import { fetchParentItems, fetchPurchaseOrders } from '../../utils/api';
+import { GoodsReceipt } from '../../interfaces';
+
+const fetchGoodsReceipt = async (id: string): Promise<GoodsReceipt> => {
+  const res = await fetch(`http://localhost:3100/api/goods-receipts/${id}`, { cache: 'no-cache' });
```

Comment:
hardcoded host here and prefer aggressive cache but for sake of this exercise no cache is fine 👍

Reply:
Refactored code to use an environment variable for the base URL. 
Yea, for the sake of the exercise, I preferred there to be no cache.